packages/terra-toggler/tests/jest/Toggler.test.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Toggler from '../../src/Toggler';
+
+describe('Toggler', () => {
+  // Snapshot Tests
+  it('should render a default toggler', () => {
+    const toggler = shallow(<Toggler>Test</Toggler>);
+    expect(toggler).toMatchSnapshot();
+  });
+
+  it('should render an animated toggler', () => {
+    const toggler = shallow(<Toggler isAnimated>Test</Toggler>);
+    expect(toggler).toMatchSnapshot();
+  });
+
+  it('should render an open toggler', () => {
+    const toggler = shallow(<Toggler isOpen>Test</Toggler>);
+    expect(toggler).toMatchSnapshot();
+  });
+
+  it('should mount a toggler', () => {
+    const toggler = mount(<Toggler>Test</Toggler>);
```

Comment:
Do we need to test Toggler with shallow, mount, and render? It seems like just one of theses should be sufficient